java/src/jmri/implementation/configurexml/MatrixSignalMastXml.java

```diff
@@ -30,7 +30,7 @@ public MatrixSignalMastXml() {
     public Element store(Object o) { // from mast p to XML
         MatrixSignalMast p = (MatrixSignalMast) o;
 
-        Element e = new Element("signalmast");
+        Element e = new Element("matrixsignalmast");
```

Comment:
This is a change the shouldn't go into a 2-9-6 versioned schema, it should instead be a 4-7-1 versioned schema, since its a completely new element, and not one that we failed to include in an earlier schema.